### DISCUSSION TITLE
Access frontmatter variable inside markdown code block

### DISCUSSION THREAD

**Discussion by ronaldaug** [vuejs/vitepress#942] [Q&A] [answered]
Is it possible to access frontmatter variable inside markdown code block? 

e.g

```
---
api_url: https://api.mydomain.com/posts
token: abcd1234567890
---

``
{{ $frontmatter.api_url }}?limit=10&token={{ $frontmatter.token  }}
``
```

But it outputs as below.
![Screenshot from 2022-07-08 12-19-19](https://user-images.githubusercontent.com/33022876/177925594-ff8c8036-0a31-4831-86f0-08d9206600ba.png)

**Comment by brc-dd** [comment, marked as answer]
#875 will fix this
